test/functional/test-xhr.js

```diff
@@ -663,6 +663,29 @@ describe('XHR', function() {
       });
     });
 
+    it('should be cloneable and each instance should provide text', () => {
+      const response = new FetchResponse(mockXhr);
+      const clone = response.clone();
+      return response.text()
+          .then(result => {
+            const callCloneText = () => clone.text().then(clonedText => {
+              expect(clonedText).to.equal(TEST_TEXT);
```

Comment:
This expectation will not be validated, since the test will have exited before this is run.